tests/materialize/fixture.json

```diff
@@ -1,4 +1,5 @@
 [
+  [],
```

Comment:
Added this initial empty transaction for unrelated reasons since I was updating all the test snapshots anyway (see commit message) - note that this causes the materialized timestamps to shift up for the subsequent transactions, which accounts for a lot of the diff in the snapshots.